src/components/home/partners/Partners.jsx

```diff
@@ -18,7 +18,6 @@ const Partners = () => {
       </Typography>
       <PrimaryPartner />
       <SecondaryPartner />
```

Comment:
Each `SecondaryPartner` on this page needs to be update to `SecondaryPartners`. It crashes otherwise.

Reply:
fixed in next push, forgot to change import when I renamed.